tests/bgp/test_traffic_shift.py

```diff
@@ -252,3 +253,79 @@ def test_TSA_B_C_with_no_neighbors(duthost, bgpmon_setup_teardown, nbrhosts, tbi
                       "Not all ipv4 routes are announced to neighbors")
         pytest_assert(wait_until(300, 3, 0, verify_all_routes_announce_to_neighs,duthost, nbrhosts, routes_6, 6),
                       "Not all ipv6 routes are announced to neighbors")
+
+def test_TSA_TSB_with_config_reload(duthost, ptfhost, nbrhosts, bgpmon_setup_teardown, traffic_shift_community):
+    """
+    Test TSA after config save and config reload
+    Verify all routes are announced to bgp monitor, and only loopback routes are announced to neighs
+    """
+    tsa_in_configdb = duthost.shell('redis-cli -n 4 hget "BGP_DEVICE_GLOBAL|STATE" tsa_enabled', module_ignore_errors=True)['stdout']
```

Comment:
We are getting the host config_db, do we need to get all the namespace config_dbs instead for multi_asic  platforms?

Reply:
Thanks for flagging this. Since the check is only for deciding whether to skip the test, added lookup in just one of the namespaces